du: add German translation (#6693)
tldr-pages/tldr: pages.de/common/du.md

# du

> Disk usage: Plattenplatzverbrauch von Dateien und Verzeichnissen ermitteln.
> Weitere Informationen: <https://www.gnu.org/software/coreutils/du>.

- Größe von Verzeichnissen und Unterverzeichnissen in den gegebenen Einheiten (B/KB/MB) auflisten:

> `du -{{b|k|m}} {{Pfad/zum/Verzeichnis}}`

Reviewer: We use `pfad/zu/verzeichnis`
```suggestion
`du -{{b|k|m}} {{pfad/zu/verzeichnis}}`
```

> - Liste die Größe von Verzeichnissen und Unterverzeichnissen in für Menschen lesbaren Einheiten auf (d. h. die automatische Auswahl der geeigneten Einheit für jede Größe):

Reviewer 2: ```suggestion
- Liste die Größe von Verzeichnissen und Unterverzeichnissen in menschenlesbaren Einheiten auf (d.h. automatische Auswahl der geeigneten Einheit für jede Größe):
```

> `du -h {{Pfad/zum/Verzeichnis}}`

Reviewer: ```suggestion
`du -h {{pfad/zu/verzeichnis}}`
```

> - Zeige die Größe eines Verzeichnisses in für Menschen lesbaren Einheiten:

Reviewer 2: ```suggestion
- Zeige die Größe eines einzelnen Verzeichnisses in menschenlesbaren Einheiten:
```

> `du -sh {{Pfad/zum/Verzeichnis}}`

Reviewer: ```suggestion
`du -sh {{pfad/zu/verzeichnis}}`
```

Author: fixed with latest commit

> - Liste die Größe eines Verzeichnisses und seiner Dateien in für Menschen lesbaren Einheiten:

Reviewer 2: ```suggestion
- Liste die Größe eines Verzeichnisses und seiner Dateien in menschenlesbaren Einheiten:
```

> `du -ah {{Pfad/zum/Verzeichnis}}`

Reviewer: ```suggestion
`du -ah {{pfad/zu/verzeichnis}}`
```

> - Auflistung der menschenlesbaren Größen eines Verzeichnisses und aller Unterverzeichnisse, bis zu einer Tiefe von `N` Ebenen:

Reviewer 2: ```suggestion
- Liste die menschenlesbaren Größen eines Verzeichnisses und aller Unterverzeichnisse, bis zu einer Tiefe von `N` Ebenen:
```

> `du -h --max-depth=N {{Pfad/zum/Verzeichnis}}`

Reviewer: ```suggestion
`du -h --max-depth=N {{pfad/zu/verzeichnis}}`
```

> - Liste die Größe aller `.jpg`-Dateien in Unterverzeichnissen des aktuellen Verzeichnisses auf, in für menschenlesbaren Einheiten und zeige am Ende die kumulierte Gesamtsumme an:

Reviewer 2: ```suggestion
- Liste die menschenlesbare Größe aller `.jpg`-Dateien in Unterverzeichnissen des aktuellen Verzeichnisses auf und zeige am Ende die kumulierte Gesamtsumme an:
```
